compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/draganddrop/DragAndDrop.jsWasm.kt

```diff
@@ -21,17 +21,14 @@ import androidx.compose.ui.geometry.Offset
 /**
  * A representation of an event sent by the platform during a drag and drop operation.
  */
-actual class DragAndDropEvent internal constructor(internal val offset: Offset)
+actual class DragAndDropEvent internal constructor(
+    internal val offset: Offset,
+    val transferData: DragAndDropTransferData?
```

Comment:
Are we sure about it? It will be public stable API with compatibility for .. forever. It sounds like it could be in `commonMain` after next iteration with a bit different signature

Also there are no docs for this public API.

Reply:
Totally agree. Made it Experimental